kedro/framework/cli/micropkg.py

```diff
@@ -854,19 +912,36 @@ def _append_package_reqs(
     )
 
 
+def _get_all_library_reqs(metadata):
+    # Inspired by https://discuss.python.org/t/\
```

Comment:
(nit) I personally find it more useful to have a short explanation of what the function does instead of just the link.

Reply:
Good point, I added a proper docstring